tidy3d/plugins/autograd/README.md

```diff
@@ -144,7 +143,12 @@ The following components are traceable as outputs of the `td.SimulationData`
   - `SimulationData.get_intensity`
   - `SimulationData.get_poynting`
 
+- Local field projections using `td.FieldProjector.project_fields()` for:
```

Comment:
was this mentioned as a "to do" item? if so would be good to remove it.

Reply:
not seeing that anywhere in this PR, but I know I've seen it somewhere...